helixbass/tree-sitter-rust
# Not intended to merge: macro invocation contents

In this PR:
- adjust Rust grammar to expose contents of macro invocations (eg `println!(...)`, `vec![...]`) as "actual Rust AST nodes"

To test:
If set up for local development, you can run `tree-sitter test` and tests should pass and if you eg run `tree-sitter parse` against a file with some "structured expression arguments" to macro invocations eg `println!(some.method_call())` then you should see the eg `call_expression` tree-sitter AST nodes nested under a `macro_invocation_contents` AST node
